contracts/AcceleratingDistributor.sol

```diff
@@ -128,7 +130,7 @@ contract AcceleratingDistributor is Testable, ReentrancyGuard, Pausable, Ownable
      * @param tokenAddress The address of the token to recover.
      * @param amount The amount of the token to recover.
      */
-    function recoverErc20(address tokenAddress, uint256 amount) external {
+    function recoverErc20(address tokenAddress, uint256 amount) external nonReentrant {
```

Comment:
reason is really that it might be useful for someone else to call it then the owner does the distribution. eg the "owner" is a multisig and then we call it from a standard EOA. However, it does not really make much of a functional difference as the caller is not the person who receives the funds. 

[this PR]( https://github.com/across-protocol/across-token/pull/8) refactors this method to behave more like a `skim` method from a uniswap pool wherein any extra tokens over and above what the contract thinks it should have. In that implementation it might make sence for us to actually send the tokens back to the caller. wdyt?

Reply:
I think we could technically allow the staking token to be included if we made this onlyOwner, which is the primary reason I was asking, but that's being considered in a separate PR. LGTM!